client/controller/api/apps.go

```diff
@@ -6,7 +6,7 @@ type App struct {
 	ID      string `json:"id"`
 	Owner   string `json:"owner"`
 	Updated string `json:"updated"`
-	URL     string `json:"url"`
+	URL     string
```

Comment:
This one I wasn't too sure about - Since the `URL` is no longer coming from the API and I am overwriting/setting it in the models within the client, do I even need to keep it in the original struct?